packages/vant/src/picker/test/index.spec.tsx

```diff
@@ -440,3 +441,53 @@ test('should render empty slot when options is empty', async () => {
   await wrapper.setProps({ columns: [{ values: ['foo'] }] });
   expect(wrapper.html()).not.toContain('empty content');
 });
+
+test('should emit correct values when clicking confirm button during column scrolling', async () => {
+  const columnsOne: PickerOption[] = [
+    { text: 'Beijing', value: 'Beijing' },
+    { text: 'Shanghai', value: 'Shanghai' },
+  ];
+  const columnsTwo: Record<string, PickerOption[]> = {
+    Beijing: [
+      { text: 'Dongcheng', value: 'Dongcheng' },
+      { text: 'Xicheng', value: 'Xicheng' },
+      { text: 'Chaoyang', value: 'Chaoyang' },
+      { text: 'Haidian', value: 'Haidian' },
+    ],
+    Shanghai: [
+      { text: 'Huangpu', value: 'Huangpu' },
+      { text: 'Xuhui', value: 'Xuhui' },
+      { text: 'Changning', value: 'Changning' },
+      { text: 'Pudong', value: 'Pudong' },
+    ],
+  };
+  const currentValues = ref(['Beijing', 'Dongcheng']);
+  const wrapper = mount({
+    setup() {
+      const columns = computed(() => [
+        columnsOne,
+        columnsTwo[currentValues.value[0]],
+      ]);
+      return () => (
+        <Picker v-model={currentValues.value} columns={columns.value} />
+      );
+    },
+  });
+  const picker = wrapper.findComponent(Picker);
+  // Scroll to select "Shanghai"
+  triggerDrag(picker.findAll('.van-picker-column')[0], 0, -100);
+  // Trigger confirm immediately without waiting for scroll animation to complete
+  // ⚠️ DO NOT uncomment, this line is for documentation purposes only
+  // await picker.find('.van-picker-column ul').trigger('transitionend');
```

Comment:
```suggestion
```
Since the comment in the previous line is very clear, we can remove the comment here.